### DISCUSSION TITLE
Tuya TS0202 TZ3000_mcxw5ehu

### DISCUSSION THREAD

**Discussion by DaywaIker313** [Koenkk/zigbee2mqtt#12480] [Devices]
I had this paired with HA via Z2M a few weeks ago and had no issues, however, now it showed up as unsupported. 

I did reconnect and reset the whole system including a fresh install (HA as OS) and since then I cannot connect the sensor anymore. 

HA is receiving messages from it but it still says it is unsupported.

Warning Received message from unsupported device with Zigbee model 'TS0202' and manufacturer name '_TZ3000_mcxw5ehu'
Warning Please see: https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html.

I tried to follow the https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html#instructions instructions to add the device as supported, but my yamal skills are not good enough to get this sorted. Tried for 3 hours but can't get it done. 

Anyone can give some advise on this?

Much appreciated.

**Comment by DaywaIker313** [comment]
Here the screenshot from HA

![image](https://user-images.githubusercontent.com/41849953/168454121-1df1c3b2-5ea7-4024-8116-58e14ca1a86d.png)

Here the error LOG

![image](https://user-images.githubusercontent.com/41849953/168454136-cb9029e9-8ddf-48ff-8deb-e22dfa655445.png)

**Reply by DaywaIker313** [reply]
Anyone some advice on this issue?

**Comment by Impact123** [comment]
My advice would be to post information about your environment. Especially the software versions you use.

**Reply by DaywaIker313** [reply]
I have the latest HA OS running installed via generic intel. 

My dongel is a Sonoff Zigbee 3.0 Plus, did not flash firmware update and used it out of the box. (All info on it below, at least all I can find)

DEVLINKS: >-
  /dev/serial/by-id/usb-ITead_Sonoff_Zigbee_3.0_USB_Dongle_Plus_4a685eb9a812ec11909d20c7bd930c07-if00-port0
  /dev/serial/by-path/pci-0000:00:14.0-usb-0:2.3:1.0-port0
DEVNAME: /dev/ttyUSB0
DEVPATH: /devices/pci0000:00/0000:00:14.0/usb2/2-2/2-2.3/2-2.3:1.0/ttyUSB0/tty/ttyUSB0
ID_BUS: usb
ID_MODEL: Sonoff_Zigbee_3.0_USB_Dongle_Plus
ID_MODEL_ENC: Sonoff\x20Zigbee\x203.0\x20USB\x20Dongle\x20Plus
ID_MODEL_ID: ea60
ID_PATH: pci-0000:00:14.0-usb-0:2.3:1.0
ID_PATH_TAG: pci-0000_00_14_0-usb-0_2_3_1_0
ID_REVISION: '0100'
ID_SERIAL: ITead_Sonoff_Zigbee_3.0_USB_Dongle_Plus_4a685eb9a812ec11909d20c7bd930c07
ID_SERIAL_SHORT: 4a685eb9a812ec11909d20c7bd930c07
ID_TYPE: generic
ID_USB_DRIVER: cp210x
ID_USB_INTERFACES: ':ff0000:'
ID_USB_INTERFACE_NUM: '00'
ID_VENDOR: ITead
ID_VENDOR_ENC: ITead
ID_VENDOR_ID: 10c4
MAJOR: '188'
MINOR: '0'
SUBSYSTEM: tty
TAGS: ':systemd:'
USEC_INITIALIZED: '3516244'

Hope that helps?

**Reply by DaywaIker313** [reply]
![image](https://user-images.githubusercontent.com/41849953/168543221-848c6246-f977-41f9-a2ad-bc7341131d34.png)
adding more info on HA

**Reply by Impact123** [reply]
What about the Zigbee2MQTT version?

**Reply by DaywaIker313** [reply]
![Screenshot_20220516-172951_Home Assistant](https://user-images.githubusercontent.com/41849953/168562902-8b72239a-4b54-4ca9-afe3-e61c74f7c698.jpg)

**Comment by DaywaIker313** [comment]
![Uploading Screenshot_20220516-172951_Home Assistant.jpg…]()

**Comment by DaywaIker313** [comment]
This is the z2m version

**Comment by Impact123** [comment]
I think support for `TZ3000_mcxw5ehu` has been added in `1.23.0`. You need to update.
I don't use HA so I can't tell you how but maybe that helps.

**Reply by DaywaIker313** [reply]
Ok let me try that

**Reply by DaywaIker313** [reply]
Much appreciated

**Reply by DaywaIker313** [reply]
legend, updated and now it is availabel again!
